src/Builder/ShowBuilder.php

```diff
@@ -32,19 +32,11 @@ public function __construct(TypeGuesserInterface $guesser, array $templates)
         $this->templates = $templates;
     }
 
-    /**
-     * @return \Sonata\AdminBundle\Admin\FieldDescriptionCollection
```

Comment:
why remove this return? better use the class name but not the fqcn

Reply:
Because it's also in the [interface](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Builder/ShowBuilderInterface.php#L26).